Examples/Io/Root/src/RootParticleWriter.cpp

```diff
@@ -8,7 +8,13 @@
 
 #include "ActsExamples/Io/Root/RootParticleWriter.hpp"
 
+#include "Acts/Definitions/TrackParametrization.hpp"
 #include "Acts/Definitions/Units.hpp"
+#include "Acts/EventData/TrackParameters.hpp"
+#include "Acts/Propagator/Propagator.hpp"
+#include "Acts/Propagator/SympyStepper.hpp"  // or EigenStepper / RKStepper
```

Comment:
```suggestion
#include "Acts/Propagator/SympyStepper.hpp"
```
I think it is fine to just rely on the sympy stepper